api/src/controllers/team.js

```diff
@@ -16,14 +21,14 @@ const {
 const { fillErrorObject } = require('../middleware/error');
 
 /**
- * Associates a twitter handle with a team on the /team/twitter-handle/:team-id endpoint.
- * @param {*} req request object, containing the team_id in the url and twitter handle in the body
- * @param {*} res response object
- * @returns 200: successful added twitter handle to team
- * @returns 400: team id is not in a valid hexadecimal format
- * @returns 404: team is not found, or handle is invalid
- * @returns 500: error trying to update the document in db
- */
+  * Associates a twitter handle with a team on the /team/twitter-handle/:team-id endpoint.
+  * @param {*} req request object, containing the team_id in the url and twitter handle in the body
+  * @param {*} res response object
+  * @returns 200: successful added twitter handle to team
+  * @returns 400: team id is not in a valid hexadecimal format
+  * @returns 404: team is not found, or handle is invalid
+  * @returns 500: error trying to update the document in db
+  */
```

Comment:
nit: maybe don't add a space between the jsdocs here. It'll inflate the diff count and I believe most of our IDE's are configured to format the `*` in the old way. Ditto for other unnecessary spaces.

Reply:
this was from the eslint fix btw!